integration/test/metrics_number_dimension/metrics_number_dimension_test.go

```diff
@@ -0,0 +1,133 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package metrics_number_dimension
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/amazon-cloudwatch-agent/integration/test"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatch/types"
+	"log"
+	"strings"
+	"testing"
+	"time"
+)
+
+const configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+const configJSON = "/config.json"
+const namespace = "MetricNumberDimensionTest"
```

Comment:
Does reusing the namespace here mess up the test? 

I know when I was doing the integration tests for CWL, I ran into a problem verifying the data that got sent because I used a shared log group/stream, and since the test gets run across different OSes in parallel, that messed up the expected number of logs.

Reply:
No this is fine. I have no issue.